app/services/research_safety_adapters/scishield_api_client.rb

```diff
@@ -11,6 +11,15 @@ class ScishieldApiClient
     PRIVATE_KEY = Rails.application.secrets.dig(:scishield, :rsa_private_key)
     API_ENDPOINT = Rails.application.secrets.dig(:scishield, :scishield_endpoint)
 
+    def invalid_response?(email)
+      response = training_api_request(email)
+      http_status_error = response.code.match?(/5|403|404/)
+      certification_data = JSON.parse(response.body)
+      certification_data_error = certification_data["data"].nil?
```

Comment:
Why not use dig here?
```suggestion
      certification_data_error = certification_data.dig("data").nil?
```

Reply:
Rubocop suggested this change. I can revert it